src/xiopak.cpp

```diff
@@ -1012,6 +1012,15 @@ void xfjoinpath(			// Joins two directory path together
 	strcpy(fullPath, directoryPath.string().c_str());
 }  /* xfjoinpath */
 //=============================================================================
+bool xfisabsolutepath(// Checks whether the path is absolute
+	const char* path) // Input path
+// Returns true if the path, in native format, is absolute, false otherwise.
+// Note: The path "/" is absolute on a POSIX OS, but is relative on Windows.
+{
+	filesys::path tempPath(path);
+	return tempPath.is_absolute();
```

Comment:
This can be a single line.

Reply:
Yeah but.

I often write "float result = blah blah blah; return result".  This enables easy inspection of "result" in the debugger in DEBUG build.  I assume that optimizer will get rid of the useless assignment in RELEASE, although I have not actually verified that.

Best choice is probably case-by-case depending on whether debug inspection looks probable.

Reply:
So that would be:
```c++
bool result = filesys::path(path).is_absolute();
return result;
```

I generally don't like unnecessary assignments, but I don't have a strong opinion on this.